site/content/en/blog/releases/1.17.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.17.0 - Kubernetes 1.20, deprecated required/preferred, add sync property to FleetAutoScaler"
+linkTitle: "1.17.0"
+date: "2021-08-31"
+---
+
+
+This is the 1.17.0 release of Agones.
+
+This release brings us support for Kubernetes 1.20, so if you are testing this release candidate, remember to upgrade your clusters.
+
+Starting from this release, required and preferred are deprecated in GameServerAllocation Specification, please use selectors for future allocation match labels. 
```

Comment:
nit: I think it would look better to surround required, preferred, and selectors with back-ticks so that they stand out from the text.